scene/gui/button.cpp

```diff
@@ -214,9 +214,10 @@ void Button::_notification(int p_what) {
 			const RID ci = get_canvas_item();
 			const Size2 size = get_size();
 
+			const Ref<StyleBox> style = _get_current_stylebox();
```

Comment:
```suggestion
			Ref<StyleBox> style = _get_current_stylebox();
```
`const` doesn't change anything with a `Ref` here so it's misleading IMO

Reply:
`const` makes sure `style` can't be reassigned. Thus making the code easier to understand especially in such a long `case` branch.

Reply:
Sure but that would be unlikely to happen, and such a change would be strange to begin with and anyone deciding to do so would just remove the `const` if they thought it was a good idea to reassign it